config.go

```diff
@@ -40,6 +41,10 @@ func ApiHost() *url.URL {
 	return host
 }
 
+func PackName() string {
```

Comment:
We can make it unexported. This method is only accessed by main.go, and both are in the same package.